x-pack/legacy/plugins/spaces/public/views/management/components/confirm_delete_modal.tsx

```diff
@@ -122,17 +122,17 @@ class ConfirmDeleteModalUI extends Component<Props, State> {
               <EuiFormRow
                 label={intl.formatMessage({
                   id: 'xpack.spaces.management.confirmDeleteModal.confirmSpaceNameFormRowLabel',
-                  defaultMessage: 'Confirm space name to delete',
+                  defaultMessage: 'Please type DELETE to delete your space.',
                 })}
                 isInvalid={!!this.state.error}
                 error={intl.formatMessage({
                   id: 'xpack.spaces.management.confirmDeleteModal.spaceNamesDoNoMatchErrorMessage',
-                  defaultMessage: 'Space names do not match.',
+                  defaultMessage: 'Incorrect input entered.',
```

Comment:
To make this a complete sentence, how about:

The input entered is incorrect. 

I'm not familiar with the context here, but it might be useful to enter what the correct input should be.